src/renderer/components/editor.tsx

```diff
@@ -32,7 +32,8 @@ export class Editor extends React.Component<EditorProps> {
   constructor(props: EditorProps) {
     super(props);
 
-    switch (props.id) {
+    const fileExtension = props.id.split('.')[1];
```

Comment:
I think this fails on custom entries that have a dot in the filename?

Another part of the code uses

`const getSuffix = (name: string) => path.parse(name).ext.slice(1);`

Also depending on how you feel about removing the `switch` we could also use `if props.id.endsWith('.html')...else`